tests/integration/test_design.py

```diff
@@ -57,6 +58,13 @@
 from ansys.geometry.core.sketch import Sketch
 
 
+# TODO: re-enable when Linux service is able to use measure tools
+def skip_if_linux(modeler: Modeler):
+    """Skip test if running on Linux."""
+    if modeler.client.backend_type == BackendType.LINUX_SERVICE:
+        pytest.skip("Measure tools not available on Linux service.")
+
+
 def test_design_extrusion_and_material_assignment(modeler: Modeler):
     """Test in charge of validating the extrusion of a simple circle as a cylinder and
     assigning materials to it."""
```

Comment:
```suggestion
    """Test to validate the extrusion of a simple circle as a cylinder and
    assign materials to it."""
```

Reply:
This is not my change at first place, but in spirit of better docs I am adding your suggestion here as well.